src/components/todo/NewTodo.tsx

```diff
@@ -0,0 +1,45 @@
+import todo from 'apis/todo';
+import { TodosContext } from 'contexts/todosContext';
+import type { ChangeEvent, FormEventHandler } from 'react';
+import { useContext, useState } from 'react';
+
+const NewTodo = () => {
+  const { setTodos } = useContext(TodosContext);
+
+  const [inputValue, setInputValue] = useState('');
+
+  const handleSubmit: FormEventHandler = async (e) => {
+    e.preventDefault();
+    todo.createTodo(inputValue).then((response) =>
+      setTodos((prev) => {
+        if (!prev) return [];
+        return [...prev, response.data];
+      }),
+    );
+    setInputValue('');
+  };
+
+  const handleChangeText = (e: ChangeEvent<HTMLInputElement>) => {
+    setInputValue(e.currentTarget.value);
+  };
+
+  return (
+    <div className="pt-16  flex flex-col items-center gap-2">
+      <form onSubmit={handleSubmit}>
```

Comment:
불필요한 depth인 것 같습니다
div태그는 스타일만 하고 있어서 form과 합치면 어떨까요? 
```suggestion
      <form className="pt-16  flex flex-col items-center gap-2" onSubmit={handleSubmit} >
```

Reply:
동의합니다.